### PR TITLE
Translate all index.md files

### PR DESCRIPTION
**Pull request by mahdiHash**

### CODE REVIEW

File: 1-js/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# The JavaScript language
+# زبان جاوااسکریپت
 
-Here we learn JavaScript, starting from scratch and go on to advanced concepts like OOP.
+اینجا ما جاوااسکریپت را یاد می‌گیریم و از ابتدا شروع می‌کنیم و تا مباحث پیشرفته مانند شیءگرایی ادامه می‌دهیم.
 
-We concentrate on the language itself here, with the minimum of environment-specific notes.
+ما اینجا، با استفاده حداقلی از نکات مربوط به محیط‌های اجرای خاص، بر روی خود زبان تمرکز می‌کنیم.
```
Comment (mahdiHash) [inline 1-js/index.md:5]: More attention here, please.

File: 2-ui/2-events/index.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
-# Introduction into Events
+# معرفی Events
 
-An introduction to browser events, event properties and handling patterns.
+معرفی eventهای مرورگر، ویژگی‌های event و مدیریت الگوها (pattern).
```
Comment (mahdiHash) [inline 2-ui/2-events/index.md:3]: More attention here.

File: 2-ui/4-forms-controls/index.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
-# Forms, controls
+# فرم‌ها، کنترل‌ها
 
-Special properties and events for forms `<form>` and controls: `<input>`, `<select>` and other.
+ویژگی‌ها و eventهای خاص برای فرم‌ها `<form>` و کنترل‌ها: `<input>`، `<select>` و دیگر موارد.
```
Comment (mahdiHash) [inline 2-ui/4-forms-controls/index.md:3]: More attention here.

File: 2-ui/5-loading/index.md [hunk 4]
```diff
@@ -1,2 +1,2 @@
 
-# Document and resource loading
+# Document و بارگیری resourceها
```
Comment (mahdiHash) [inline 2-ui/5-loading/index.md:2]: Should "resources" be translated?

File: 8-web-components/index.md [hunk 5]
```diff
@@ -1,3 +1,3 @@
-# Web components
+# اجزاء وب
 
-Web components is a set of standards to make self-contained components: custom HTML-elements with their own properties and methods, encapsulated DOM and styles.
+اجزاء وب (web component) مجموعه‌ای از استانداردها هستند تا اجزای self-contained بسازند: المان‌های HTML شخصی‌سازی شده با ویژگی‌ها و متدهای خودشان، DOM کپسوله و استایل‌ها.
```
Comment (mahdiHash) [inline 8-web-components/index.md:3]: More attention here, please.

File: 9-regular-expressions/index.md [hunk 6]
```diff
@@ -1,3 +1,3 @@
-# Regular expressions
+# Regular expressionها
 
-Regular expressions is a powerful way of doing search and replace in strings.
+Regular expressionها راهی قدرتمند برای انجام جستجو و جایگزین کردن در رشته‌ها هستند.
```
Comment (mahdiHash) [inline 9-regular-expressions/index.md:3]: "Regular expressions" --> "عبارات با قاعده" ?

File: 2-ui/5-loading/index.md (hunk 4 above)
Comment (mahdyar) [reply 2-ui/5-loading/index.md:2]: Yes.

File: 9-regular-expressions/index.md (hunk 6 above)
Comment (mahdyar) [reply 9-regular-expressions/index.md:3]: > "Regular expressions" --> "عبارات باقاعده" ?

Yes/

**Comment by mahdiHash** [comment]
Thank you @mahdyar.

**Pull request merged**